doc/man3/EVP_DigestInit.pod

```diff
@@ -180,18 +180,29 @@ See L</PARAMETERS> below for more information.
 Sets the list of I<params> into a MD context I<ctx>.
 See L</PARAMETERS> below for more information.
 
-=item EVP_MD_gettable_params(), EVP_MD_gettable_ctx_params(),
-EVP_MD_settable_ctx_params(), EVP_MD_CTX_gettable_params(),
-EVP_MD_CTX_settable_params()
-
-Get a B<OSSL_PARAM> array that describes the retrievable and settable
-parameters. EVP_MD_gettable_params() returns parameters that can be used with
-EVP_MD_get_params(). EVP_MD_gettable_ctx_params() and
-EVP_MD_CTX_gettable_params() return parameters that can be used with
-EVP_MD_CTX_get_params(). EVP_MD_settable_ctx_params() and
-EVP_MD_CTX_settable_params() return parameters that can be used with
-EVP_MD_CTX_set_params().
-See L<OSSL_PARAM(3)> for the use of B<OSSL_PARAM> as parameter descriptor.
+=item EVP_MD_gettable_params()
+
+Get a constant B<OSSL_PARAM> array that describes the retrievable parameters
+that can be used with EVP_MD_get_params().  See L<OSSL_PARAM(3)> for the
+use of B<OSSL_PARAM> as a parameter descriptor.
+
+=item EVP_MD_gettable_ctx_params(), EVP_MD_CTX_gettable_params()
```

Comment:
These should all be in the history section

Reply:
Plus some others....